src/components/Footer.tsx

```diff
@@ -0,0 +1,51 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterName } from '../types/enumFilterName';
+
+type Props = {
+  activeTodos: number;
+  filteredBy: FilterName;
+  setFilteredBy: (FilterName: FilterName) => void;
+  todos: Todo[];
+  deleteCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  filteredBy,
+  setFilteredBy,
+  todos,
+  deleteCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {[FilterName.ALL, FilterName.ACTIVE, FilterName.COMPLETED].map(link => (
```

Comment:
You can use here {Object.values(FilterName).map(filterItem => (

Reply:
> You can use here {Object.values(FilterName).map(filterItem => (

Good point! 🔥 
Do it